cal/cal/templates/home_logged_in.html

```diff
@@ -26,6 +26,13 @@
         }
       });
     });
+    function runFullSync() {
+      if ("{{request.user.profile.first_time_authed}}" == "True") {
+        "{{request.user.profile.resetFirstTimeAuth}}";
```

Comment:
I think you are just referencing the function here instead of calling it. I might be wrong though since I haven't verified it myself.